concent_api/core/message_handlers.py

```diff
@@ -389,6 +389,9 @@ def handle_send_force_get_task_result(client_message: message.concents.ForceGetT
         task_to_compute,
     )
 
+    if Subtask.objects.filter(subtask_id=task_to_compute.compute_task_def['subtask_id']).exists():  # pylint: disable=no-member
+        Subtask.objects.select_for_update().get(subtask_id=task_to_compute.compute_task_def['subtask_id'])  # pylint: disable=no-member
```

Comment:
I'd replace this with `try: subtask = Subtask.objects.select_for_update().get(...) except DoesNotExist: ...` construct.

Why make two queries when you can get away with making just one.

Reply:
done